app/invocation/invocation.css

```
@@ -632,13 +632,18 @@
   flex-grow: 10;
 }
 
-.invocation-share-button svg {
+.invocation-share-button svg,
+.invocation-ask-button svg {
   margin-right: 12px;
   margin-left: -4px;
   width: 16px;
   height: 16px;
 }
 
+.invocation-ask-button {
+  background-color: #2196f3 !important;
```

nit: would this work instead of `!important` ?

```suggestion
.filled-button.invocation-ask-button {
  background-color: #2196f3;
```

---

Ah yeah, this was me being lazy during hack-week. Fixed!